apis/management.cattle.io/v3/cluster_template_types.go

```diff
@@ -0,0 +1,62 @@
+package v3
+
+import (
+	"github.com/rancher/norman/types"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type ClusterTemplate struct {
+	types.Namespaced
+
+	metav1.TypeMeta `json:",inline"`
+	// Standard object’s metadata. More info:
+	// https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#metadata
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec ClusterTemplateSpec `json:"spec"`
+}
+
+type ClusterTemplateSpec struct {
+	DisplayName         string `json:"displayName" norman:"required"`
+	Description         string `json:"description"`
+	Disabled            bool   `json:"disabled,omitempty"`
+	DefaultRevisionName string `json:"defaultRevisionName,omitempty" norman:"type=reference[clusterTemplateRevision]"`
+
+	Members       []Member `json:"members,omitempty"`
+	RevisionLimit int      `json:"revisionLimit,omitempty" norman:"default=10"`
```

Comment:
We might not need it anymore as create/delete of revisions is controlled via API

Reply:
Will remove this limit.